Assets/Shopify/UIToolkit/Themes/Generic/GenericSingleProductTheme.cs

```diff
@@ -81,6 +85,29 @@ public class GenericSingleProductTheme : MonoBehaviour, ISingleProductTheme {
             UpdateDetailsUsingVariant(wrappedVariant.ProductVariant);
         }
 
+        public void SelectImage(Button button) {
+            ActiveImage.GetComponent<UnityEngine.UI.Image>().sprite = button.image.sprite;
+        }
+
+        private void SetupPreviewImages(Product product) {
+            var images = (List<Shopify.Unity.Image>)product.images();
+            if (images.Count == 0) {
+                // TODO: Hide image area?
```

Comment:
we should display a placeholder in this case imo

Reply:
Could we also disable the container that holds all the images and have the product text just expand full width?

Reply:
might be too wide... future problems lol